.env

```
@@ -35,3 +35,6 @@ EMAIL_CONTACT=contact@print4health.org
 ###> sentry/sentry-symfony ###
 SENTRY_DSN=
 ###< sentry/sentry-symfony ###
+###> google/geocoding service ###
+GOOGLE_MAPS_API_KEY=
```

```suggestion
GOOGLE_MAPS_API_KEY=''
```
not sure about this but I think it should be an empty string?